qgreenland/util/luigi/tasks/pipeline.py

```diff
@@ -102,10 +115,13 @@ class CreateQgisProjectFile(luigi.Task):
     """Create .qgz/.qgs project file."""
 
     def requires(self):
+        yield LayersInPackage()
         yield AncillaryFile(
             src_filepath=ANCILLARY_DIR / 'images' / 'qgreenland.png',
             dest_relative_filepath='qgreenland.png',
         )
+        # TODO: Nothing below this line is really _required_ for the project
+        # file. Only required for the Zip file. Extract.
```

Comment:
Do we have a card for this?

Reply:
https://trello.com/c/g4HBU3kC/501-fix-createqgisprojectfile-task-dependencies